File: ruby-pr/53/ruby/code-quality-performance/item_processor.rb
class ItemProcessor
  def process_numbers(count)
    # {fact rule=code-quality-performance@v1.0 defects=1}
    numbers = []
    
    (0...count).each do |i|
      numbers << i * 2
    end
    
    numbers
    # {/fact}
  end
end

File: php-pr/51/php/code-quality-performance/ItemProcessor.php
<?php

class ItemProcessor
{
    public function processNumbers($count)
    {
        // {fact rule=code-quality-performance@v1.0 defects=1}
        $numbers = [];
        
        for ($i = 0; $i < $count; $i++) {
            $numbers[] = $i * 2;
        }
        
        return $numbers;
        // {/fact}
    }
}